src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.MetadataUpdate.cs

```diff
@@ -378,11 +378,28 @@ private static void ReplaceViewInstance(UIElement instance, Type replacementType
 
 	public static void UpdateApplication(Type[] types)
 	{
-		foreach (var t in types)
+		foreach (var type in types)
 		{
-			if (t.GetCustomAttribute<System.Runtime.CompilerServices.MetadataUpdateOriginalTypeAttribute>() is { } update)
+			try
 			{
-				TypeMappings.RegisterMapping(t, update.OriginalType);
+				// Look up the attribute by name rather than by type.
+				// This would allow netstandard targeting libraries to define their own copy without having to cross-compile.
+				var attr = type.GetCustomAttributesData().FirstOrDefault(data => data is { AttributeType.FullName: "System.Runtime.CompilerServices.MetadataUpdateOriginalTypeAttribute" });
```

Comment:
This will only work when doing hot reload with the devserver, VS will not use this.